pom.xml

```diff
@@ -387,6 +387,9 @@
         <surefire.jvm.args>${surefire.jvm.args.memory} ${surefire.jvm.args.java-version} ${surefire.jvm.args.jacoco}</surefire.jvm.args>
         <failsafe.jvm.args>${surefire.jvm.args.memory} ${surefire.jvm.args.java-version} ${failsafe.jvm.args.jacoco}</failsafe.jvm.args>
 
+        <failsafe.wildfly.skip>false</failsafe.wildfly.skip>
+        <failsafe.osgi.skip>false</failsafe.osgi.skip>
```

Comment:
It seems you forgot to set them to true for the JDK 12+ profile? Or did I miss something?

Reply:
I did that in the Jenkinsfile. This way, if someone tries to build by hand, they will see that it doesn't work (yet) on WF and OSGi, but our CI tests will still check everything else.

Reply:
Hum. Not sure it's the desired behavior. That means people won't be able to build our code with the latest JDK.

Reply:
That was not my goal at all... but ok, I'll do it your way.